client/webserver/site/src/js/markets.ts

```diff
@@ -469,6 +466,9 @@ export default class MarketsPage extends BasePage {
         this.setMarket(row.mkt.xc.host, row.mkt.baseid, row.mkt.quoteid)
       })
     }
+    if (!State.showLeftMarketDock()) { // It is shown by default, hiding if necessary.
+      closeMarketsList()
+    }
```

Comment:
Alternatively we can generate template without left-market-doc on backend, but doing it like this was just easier.
How do you choose between using back-end-templating / javascript for things like these ? Here it doesn't seem to make any difference.

Reply:
Also, if we leave it in JS, I believe [local storage can be used instead of cookies](https://stackoverflow.com/questions/3220660/local-storage-vs-cookies), what do you think ?

Reply:
signOut presently does `window.localStorage.clear()` for security reasons but I suppose that could change.

Reply:
> Alternatively we can generate template without left-market-doc on backend, but doing it like this was just easier. How do you choose between using back-end-templating / javascript for things like these ? Here it doesn't seem to make any difference.

It's better how you have it.

Reply:
> It's better how you have it.

What about cookies vs local storage ? I think I should change to local storage.

Reply:
> > It's better how you have it.
> 
> What about cookies vs local storage ? I think I should change to local storage.

I suppose anything the server doesn't use would be better off in local storage.

Reply:
Moved to local storage.

Reply:
So it's meant to be lost on logout?

Reply:
Oh man, I forgot about your comment above! Gotta address that, we don't want to loose it on logout - [changed](https://github.com/decred/dcrdex/pull/1975/commits/11b05754d7dc905ce8baec53b9985ec46a7819ae) and re-tested (seems to be working).

So, now on logout only `dexauth` cookie gets cleaned up, I understand this is the behavior that fits best.

Reply:
> How do you choose between using back-end-templating / javascript for things like these ?

When there isn't a clear choice, I lean Javascript.